### DISCUSSION TITLE
How can I move the canvas with the secondary button on my Wacom Pen?

### DISCUSSION THREAD

**Discussion by BusterLundgaard** [flxzt/rnote#764] [Q&A] [answered]
Hi! I'm trying to set up RNote so that i can pan around on the canvas by holding down the secondary button on my pen and dragging the canvas. Mapping the secondary button on my pen to either space or middle mouse button works in other programs, but not RNote. What can I do to get this functionality? 
I'm using a Cintique Pro 13 Model DTH-1320. 
Thanks a lot for the awesome program!

**Comment by flxzt** [comment]
If the button works in general on Wayland, you could do: map the button inside the app to `Tools` (the wrench symbol). Then select the `move view` tool in the sidebar. When you press the button it should switch to it , and it should be remembered across activations/app restarts.

**Comment by BusterLundgaard** [comment, marked as answer]
Thanks @flxzt. 

Also found out just now that it works to just set the secondary button to the `ALT` modifier key.

**Reply by flxzt** [reply]
I see, that is a good solution as well  :+1:
